modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/router/QueueMessageRouter.java

```diff
@@ -60,7 +60,7 @@ public QueueMessageRouter(String encodedRouterInfo) {
      * @throws AndesException when there are active subscriptions
      */
     public void removeMapping(String bindingKey, StorageQueue queue) throws AndesException {
-        if (queue.getBoundedSubscriptions().isEmpty()) {
+        if (queue.getBoundSubscriptions().isEmpty()) {
```

Comment:
Can 'queue.getBoundSubscriptions()' be actually empty when this method is called?